rpc/rpc.go

```diff
@@ -591,7 +591,7 @@ func (c *Conn) handleBootstrap(ctx context.Context, id answerID) error {
 		syncutil.Without(&c.mu, rl.release)
 		return nil
 	}
-	if err := ans.setBootstrap(c.bootstrap.AddRef()); err != nil {
+	if err := ans.setBootstrap(c.bootstrap); err != nil {
```

Comment:
I'm still not convinced this is correct; my intuition is that this *should* be needed, since we want the caller to get their own reference, rather than stealing ours.

It makes sense that this fixes it given that you removed a Release() elsewhere, but I want to make sure there's not just another failure to call .Release() somewhere that's cancelling this out.

If this is correct I think it's important for us to be able to articulate why.